ui/src/components/Settings/Plex/index.tsx

```diff
@@ -213,6 +213,8 @@ const PlexSettings = () => {
   const verifyToken = (token?: string) => {
     const authToken = token || settingsCtx.settings.plex_auth_token
     if (authToken) {
+      const source = axios.CancelToken.source()
```

Comment:
This method of cancelling is deprecated, AbortController is the way to go now:

```suggestion
      const controller = new AbortController();
```

You also need to pass this into axios for it to work